web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -929,3 +929,17 @@ function joinup_core_form_views_exposed_form_alter(&$form, FormStateInterface $f
 function joinup_core_preprocess_page(&$variables) {
   $variables['#attached']['library'][] = 'joinup_core/sign_in_destination';
 }
+
+/**
+ * Implements hook_editor_js_settings_alter().
+ *
+ * Strip off the h1, h5, h6 paragraph format options from the 'Format' dropdown.
```

Comment:
Swap "paragraph" here with tags, as they actually are tags.

Reply:
I changed it with 'tag' as the plural goes to options.